metrics-scala/src/main/scala/nl/grons/metrics4/scala/MetricName.scala

```diff
@@ -82,7 +82,8 @@ class MetricName private (val name: String) {
 
     val sb = new StringBuilder(name)
     names.view
-      .filterNot(StringUtils.isEmpty)
+      .filter(_ != null)
+      .filter(_.nonEmpty)
```

Comment:
Its probably faster if we combine these.

```scala
 .filter(n => n != null && n.nonEmpty)
```

Reply:
Done :)